{"uuid": "80b5cb80-6497-4acc-807f-ce92375c97aa", "counter": 6, "stdout": "\u001b[0;31mfatal: [192.168.38.9]: FAILED! => {\"ansible_facts\": {\"discovered_interpreter_python\": \"/usr/bin/python3\"}, \"changed\": true, \"cmd\": [\"ip\", \"netns\", \"add\", \"sa_ge\"], \"delta\": \"0:00:00.010156\", \"end\": \"2024-04-23 23:14:57.641715\", \"msg\": \"non-zero return code\", \"rc\": 1, \"start\": \"2024-04-23 23:14:57.631559\", \"stderr\": \"Cannot create namespace file \\\"/run/netns/sa_ge\\\": File exists\", \"stderr_lines\": [\"Cannot create namespace file \\\"/run/netns/sa_ge\\\": File exists\"], \"stdout\": \"\", \"stdout_lines\": []}\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "b8e70068-de09-49d9-9028-84ec34e87ff9", "event": "runner_on_failed", "pid": 178094, "created": "2024-04-23T23:14:57.713610", "parent_uuid": "52540038-3809-3f1c-c214-000000000004", "event_data": {"playbook": "/home/vmadm/demo/RapidServe/southbound/deploy_vpc.yaml", "playbook_uuid": "3aa07111-8724-4af7-9be6-d5f2df3941c4", "play": "Deploy VPC Infrastructure with network namespace", "play_uuid": "52540038-3809-3f1c-c214-000000000002", "play_pattern": "hostvms", "task": "Create namespaces for each VPC", "task_uuid": "52540038-3809-3f1c-c214-000000000004", "task_action": "command", "resolved_action": "ansible.builtin.command", "task_args": "", "task_path": "/home/vmadm/demo/RapidServe/southbound/deploy_vpc.yaml:7", "host": "192.168.38.9", "remote_addr": "192.168.38.9", "res": {"changed": true, "stdout": "", "stderr": "Cannot create namespace file \"/run/netns/sa_ge\": File exists", "rc": 1, "cmd": ["ip", "netns", "add", "sa_ge"], "start": "2024-04-23 23:14:57.631559", "end": "2024-04-23 23:14:57.641715", "delta": "0:00:00.010156", "msg": "non-zero return code", "invocation": {"module_args": {"_raw_params": "ip netns add sa_ge", "_uses_shell": false, "stdin_add_newline": true, "strip_empty_ends": true, "argv": null, "chdir": null, "executable": null, "creates": null, "removes": null, "stdin": null}}, "stdout_lines": [], "stderr_lines": ["Cannot create namespace file \"/run/netns/sa_ge\": File exists"], "ansible_facts": {"discovered_interpreter_python": "/usr/bin/python3"}, "_ansible_no_log": null}, "start": "2024-04-23T23:14:56.117574", "end": "2024-04-23T23:14:57.713258", "duration": 1.595684, "ignore_errors": null, "event_loop": null, "uuid": "80b5cb80-6497-4acc-807f-ce92375c97aa"}}